tgstation.dme

```diff
@@ -3328,12 +3334,10 @@
 #include "fulp_modules\mentorhelp\mentorpm.dm"
 #include "fulp_modules\mentorhelp\mentorsay.dm"
 #include "fulp_modules\mentorhelp\mentorwho.dm"
-#include "fulp_modules\jobs\job_integration.dm"
-#include "fulp_modules\jobs\brigdoc\code\brigdoc.dm"
-#include "fulp_modules\jobs\brigdoc\code\brigdoc_clothing.dm"
-#include "fulp_modules\jobs\brigdoc\code\gamemode_overwrite.dm"
-#include "fulp_modules\jobs\brigdoc\code\landmark.dm"
 #include "fulp_modules\mob_spawners\code\mobspawner.dm"
+#include "fulp_modules\things\food\code\food.dm"
+#include "fulp_modules\things\food\code\recipe.dm"
+#include "fulp_modules\things\toys\code\toys.dm"
```

Comment:
Should probably remove the "things" folder altogether, and just have food and toys.

Reply:
yeah i was going to change the name of the folder lol